DuckDuckGo/ConfigurationDebugViewController.swift

```diff
@@ -0,0 +1,163 @@
+        return Sections.allCases.count
+    }
+
+    override func tableView(_ tableView: UITableView, titleForHeaderInSection section: Int) -> String? {
+        guard let section = Sections(rawValue: section) else { return nil }
+        return titles[section]
+    }
+
+    override func tableView(_ tableView: UITableView, cellForRowAt indexPath: IndexPath) -> UITableViewCell {
+        let cell = tableView.dequeueReusableCell(withIdentifier: "Cell", for: indexPath)
+        switch Sections(rawValue: indexPath.section) {
+
+        case .refreshInformation:
+            switch RefreshInformationRows(rawValue: indexPath.row) {
+            case .lastRefreshDate:
+                cell.textLabel?.text = "Last Refresh"
+
+                if lastConfigurationRefreshDate == Date.distantPast {
+                    cell.detailTextLabel?.text = "Never"
+                } else {
+                    cell.detailTextLabel?.text = dateFormatter.string(from: lastConfigurationRefreshDate)
+                }
+            case .resetLastRefreshDate:
+                cell.textLabel?.text = "Reset Last Refresh Date"
+            default: break
+            }
+
+        case .queuedTasks:
+            if queuedTasks.isEmpty {
+                cell.textLabel?.text = "None"
+            } else {
+                cell.textLabel?.text = dateFormatter.string(from: queuedTasks[indexPath.row].earliestBeginDate!)
+            }
+
+        case .etags:
+            let row = ETagRows.allCases[indexPath.row]
+            cell.textLabel?.text = row.rawValue
+
+            if let etag = etag(for: row) {
+                cell.detailTextLabel?.text = etag
+            } else {
+                cell.detailTextLabel?.text = "None"
+            }
+
+        default: break
+        }
+
+        return cell
+    }
+
+    override func tableView(_ tableView: UITableView, numberOfRowsInSection section: Int) -> Int {
+        switch Sections(rawValue: section) {
+        case .refreshInformation: return RefreshInformationRows.allCases.count
+        case .queuedTasks: return queuedTasks.isEmpty ? 1 : queuedTasks.count
+        case .etags: return ETagRows.allCases.count
+        default: return 0
+        }
+    }
+
+    override func tableView(_ tableView: UITableView, didSelectRowAt indexPath: IndexPath) {
+        switch Sections(rawValue: indexPath.section) {
+        case .refreshInformation:
+            switch RefreshInformationRows(rawValue: indexPath.row) {
+            case .resetLastRefreshDate:
+                lastConfigurationRefreshDate = Date.distantPast
+                tableView.reloadData()
+            default: break
+            }
+        default: break
+        }
+    }
+
+    // MARK: - ETag User Defaults Wrapper
+
+    lazy var defaults = UserDefaults(suiteName: "com.duckduckgo.blocker-list.etags")
+
+    private func etag(for config: ETagRows) -> String? {
+        return defaults?.string(forKey: config.rawValue)
+    }
```

Comment:
This isn't ideal, but to use `UserDefaultsETagStorage` directly involves making a ton of stuff in `Core` public. A few lines of duplicated code is the better approach IMO since this is just a debug menu.